test/lib/cli/update-notifier.js

```diff
@@ -83,6 +120,15 @@ const runUpdateNotifier = async (t, {
     prefixDir,
     argv,
   })
+  const registry = new MockRegistry({
+    tap: t,
+    registry: mock.npm.config.get('registry'),
+  })
+
+  if (PACOTE_MOCK_REQ_COUNT > 0) {
+    registry.nock.get('/npm').times(PACOTE_MOCK_REQ_COUNT).reply(200, pacumentResponse)
```

Comment:
This is a great iteration.  Eventually we'd want this in the mock registry (this request AND the packument it generates) but I don't want to block the PR on that.